test/functional/functional-tests.js

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright 2023 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of iotagent-json
+ *
+ * iotagent-json is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * iotagent-json is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with iotagent-json.
+ * If not, seehttp://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License
+ * please contact with::[contacto@tid.es]
+ *
+ * Modified by: Miguel Angel Pedraza
+ */
+
+/* eslint-disable no-unused-vars */
+
+const iotaJson = require('../..');
+const config = require('./config-test.js');
+const nock = require('nock');
+const iotAgentLib = require('iotagent-node-lib');
+const should = require('should');
+const async = require('async');
+
+const utils = require('../utils');
+const request = utils.request;
+
+const logger = require('logops');
+
+let contextBrokerMock;
+
+describe('FUNCTIONAL TESTS', function () {
+    beforeEach(function (done) {
+        nock.cleanAll();
+        iotaJson.start(config, function () {
+            done();
+        });
+    });
+
+    afterEach(function (done) {
+        nock.cleanAll();
+
+        async.series([iotAgentLib.clearAll, iotaJson.stop], done);
+    });
+
+    describe('Basic group provision without attributes', function () {
+        // Provision
```

Comment:
unnecessary comment

Reply:
Fixed in https://github.com/telefonicaid/iotagent-json/pull/769/commits/0a8d8c21943bdc2c8fa8564b3e109ae17f08b94a